staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/types.go

```diff
@@ -418,6 +418,11 @@ const (
 	DeletePropagationForeground DeletionPropagation = "Foreground"
 )
 
+const (
+	// DryRunTrue means that modifications will not be persisted. No other value is allowed.
```

Comment:
Suggest "DryRunTrue means that all dry run stages will be processed."

Modifications not being persisted isn't unique to this dry run directive.

I don't particularly see the need to forbid additional directives. We'll validate we recognize all the directives they specify, and the check to dry-run stage "Foo" will be if "True" or "Foo" is in the list of directives. If they specify ["True","Foo"], that is unnecessary but unambiguous.